src/components/Reviews.jsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from "react";
+import { getReviews } from "../api";
+
+function Reviews() {
+  const [reviews, setReviews] = useState([]);
+  useEffect(() => {
+    return getReviews().then(({ reviews }) => {
+      setReviews(reviews);
```

Comment:
Could you handle the loading as well please? Need some feedback something is happening if the server or internet is slow